commonmark/src/Commonmark/Inlines.hs

```diff
@@ -734,15 +738,15 @@ moveRight (Cursor (Just x) zs (y:ys)) = Cursor (Just y) (x:zs) ys
 {-# INLINE moveRight #-}
 
 processBs :: IsInline a
-          => [BracketedSpec a] -> DState a -> [Chunk a]
+          => [BracketedSpec a] -> DState a -> Either (DState a) [Chunk a]
```

Comment:
Worth a comment at the beginning explaining what Left and Right results mean here.